### DISCUSSION TITLE
Decompression problem

### DISCUSSION THREAD

**Discussion by HelloMeisn** [MustardChef/WSABuilds#247] [General]
I cannot start the WsaSettings.exe in the installation directory. When double click the exe file, it warns there's no the dll files below. And the problem seems to come from decompression. I use nanazip and winrar, but neither works.
<img width="359" alt="屏幕截图 2023-12-17 012917" src="https://github.com/MustardChef/WSABuilds/assets/148852511/69e40103-f9be-49bd-887d-66cc6f676886">
<img width="408" alt="屏幕截图 2023-12-17 012936" src="https://github.com/MustardChef/WSABuilds/assets/148852511/a8af2c38-a5c8-4536-98c2-a2937e588a4e">
<img width="406" alt="屏幕截图 2023-12-17 012949" src="https://github.com/MustardChef/WSABuilds/assets/148852511/025f6120-313a-47a0-b4be-b58a87abd4c0">
<img width="423" alt="屏幕截图 2023-12-17 013036" src="https://github.com/MustardChef/WSABuilds/assets/148852511/2f0a96cc-f0ff-4b2b-85b2-f0424959fcdb">

**Comment by MustardChef** [comment]
@HelloMeisn Please use 7-zip or update WinRAR and try again.
